### ISSUE TITLE
Toggle state is not read by screen readers in IE 11 Windows 10

### ISSUE THREAD

**Issue by PoojaOtari** [OfficeDev/office-ui-fabric-react#11566]
Environment Information
React fabric Package version(s): both versions 6.210.3 and 7.56.1 and tried others as well
Browser and OS versions: IE 11, Windows 10
Describe the issue:
Toggle state (on, off, changes in state) is not read by screen readers in IE 11.

Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fabric#/controls/web/toggle
To repro, open the fabric controls demo page in IE 11 and use Narrator while navigating to the toggle buttons. If you change the role to "checkbox" in DOM, Narrator will announce state

Actual behavior:
Narrator (or NVDA) won't announce the state of the toggle button or the changes in state.

Expected behavior:
Narrator (or NVDA) should announce the state of the toggle button or the changes in state.

Documentation describing the expected behavior
Toggles are not accessible if the user doesn't know if it's on or off.
https://www.w3.org/TR/wai-aria-1.1/#switch

Previously there was bug raised as below, according to that it should be resolved but I still see the issue with a toggle button in IE with a narrator
https://github.com/OfficeDev/office-ui-fabric-react/issues/10966

**Comment by micahgodbolt** [comment]
#10967 provides an IE11 compatible version that you can enable by setting `role="checkbox"` as you suggested.

It will be up to the app team to check for IE11 and use the checkbox role. 

```
<Toggle
        label="Enabled and checked (ARIA 1.0 compatible)"
        defaultChecked
        onText="On"
        offText="Off"
        onChange={_onChange}
        role="checkbox"
      />
```
